### PR TITLE
[TSC 2021July] Revise GOVERNANCE: Maintainer Activities

### PR DESCRIPTION
**Pull request by cognifloyd**
In the July 2021 TSC meeting, there was discussion about harmonizing the GOVERNANCE document with how maintainers currently do things. This is a quick attempt to capture those thoughts to facilitate further discussion.

### CODE REVIEW

**Review by punkrokk — APPROVED** [review]

File: GOVERNANCE.md [hunk 1]
```diff
@@ -26,8 +26,7 @@ StackStorm uses a three-tiered system of Maintainer roles:
   * Have full member write access to [StackStorm](https://github.com/stackstorm/) and [StackStorm-Exchange](https://github.com/stackstorm-exchange) Github organizations, CI/CD, Moderator at [forum](https://forum.stackstorm.com/), [Slack](https://stackstorm.com/community-signup) and other Community platforms.
   * Receive **one vote** in the voting process.
 
-### Maintainer responsibilities
-* In general dedicate at least 1+ full day per week (summarized) on StackStorm development.
+### Maintainer Activities
 * Monitor Slack, Forums and other Communication channels with StackStorm Community (delayed response is perfectly acceptable).
```
Comment (blag) [inline GOVERNANCE.md:30]:
```suggestion
* Monitor one of: Slack, Forums, and other Communication channels with StackStorm Community (delayed response is perfectly acceptable).
```

**Review by blag — APPROVED** [review]
We discussed one additional tweak (see my suggestion), but this is also good to go with or without it.

**Review by amanda11 — APPROVED** [review]

**Review by Kami — APPROVED** [review]
+1

**Review by armab — APPROVED** [review]
Thanks for the proposal change!

Cross-linking with the https://github.com/StackStorm/discussions/issues/85#issuecomment-892103104

**Review by mickmcgrath13 — APPROVED** [review]

File: GOVERNANCE.md [hunk 2]
```diff
@@ -26,9 +26,8 @@ StackStorm uses a three-tiered system of Maintainer roles:
   * Have full member write access to [StackStorm](https://github.com/stackstorm/) and [StackStorm-Exchange](https://github.com/stackstorm-exchange) Github organizations, CI/CD, Moderator at [forum](https://forum.stackstorm.com/), [Slack](https://stackstorm.com/community-signup) and other Community platforms.
   * Receive **one vote** in the voting process.
 
-### Maintainer responsibilities
-* In general dedicate at least 1+ full day per week (summarized) on StackStorm development.
-* Monitor Slack, Forums and other Communication channels with StackStorm Community (delayed response is perfectly acceptable).
+### Maintainer Activities
+* Monitor one of: Slack, Forums, and other Communication channels with StackStorm Community (delayed response is perfectly acceptable).
```
Comment (winem) [inline GOVERNANCE.md:30]: What about `one or more of` ? To me `one of` sounds like a hard limit but I guess it's in our interest to keep that up to each maintainer themselves. 

But it looks good enough to me and shouldn't prevent the PR from being merged.

**Review by winem — APPROVED** [review]

File: GOVERNANCE.md (hunk 2 above)
Comment (punkrokk) [reply GOVERNANCE.md:30]: makes sense

Comment (cognifloyd) [reply GOVERNANCE.md:30]:
```suggestion
* Monitor one or more of: Slack, Forums, and other Communication channels with StackStorm Community (delayed response is perfectly acceptable).
```
Like this?

**Review by m4dcoder — APPROVED** [review]

**Comment by cognifloyd** [comment]
By my count, after 1 week:
8/10 TSC members have approved this, and 2 have not voted.

- Leaders (3 votes each; 1 member):
  **+0/3** points
- Senior Maintainers (2 votes each; 5 members):
  **+8/10** points
- Maintainers (1 vote each; 4 members):
  **+4/4** points

Total: **+12/17** points (70.6%) to approve (5 points not voted; no negative votes). That is more than 2/3.

So, what's the next step?

**Comment by armab** [comment]
@cognifloyd Your calculations are :100: and this PR is ready to go after 7 day voting period.
Let's merge it on Monday to minimize the weekend noise for the people around TSC decisions.

**Review by blag — APPROVED** [review]

**Comment by punkrokk** [comment]
so much Sunday noise :p

JP Bourget / @punkrokk

**Pull request merged**